manifests/o/OpenShopChannel/Downloader/1.2.11/OpenShopChannel.Downloader.yaml

```
@@ -0,0 +1,21 @@
+PackageIdentifier: OpenShopChannel.Downloader
+Publisher: Open Shop Channel
```

```suggestion
Publisher: dhtdht020, Open Shop Channel
```
As per the latest documentation for Windows Package Manager, the publisher now needs to match whatever is shown in Control Panel so the application can map properly in `winget list` and allow other people to upgrade via the `winget upgrade` command.